superset/constraints-3.1.0.txt

```diff
@@ -98,7 +98,8 @@ flask==2.2.5
     #   flask-session
     #   flask-sqlalchemy
     #   flask-wtf
-flask-appbuilder==4.3.10
```

Comment:
I think we should record the original version (that we have bumped from) in the comments as otherwise we have no way of knowing what was there (as the source contraints files are not versioned).

Reply:
(OK, we have our own repo history, but this would make it quicker to find when trouble-shooting!)

Reply:
Done https://github.com/stackabletech/docker-images/pull/686/commits/6aba8d7a2b9d5d7e936c3d27583a0c092dc4862e

Reply:
Thanks!